examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -213,6 +213,12 @@ def __init__(
             self.gui_input,
         )
 
+        # HACK: Available actions are hardcoded by agent type.
+        #       Only humans can change object states.
+        can_change_object_states = isinstance(
+            self.gui_agent_controller, GuiHumanoidController
+        )
```

Comment:
@zephirefaith : I intend to move this into an `rearrange_v2`-level config, along with head sensor definition and feature flags.